gnat2goto/driver/tree_walk.adb

```diff
@@ -1394,8 +1394,8 @@ package body Tree_Walk is
 
    function Do_String_Constant (N : Node_Id) return Irep is
       Ret              : constant Irep := New_Irep (I_String_Constant_Expr);
-      Element_Type_Ent : constant Entity_Id := Get_Array_Component_Type (N);
-      Element_Type     : constant Irep := Do_Type_Reference (Element_Type_Ent);
+     -- Element_Type_Ent : constant Entity_Id := Get_Array_Component_Type (N);
+     -- Element_Type  : constant Irep := Do_Type_Reference (Element_Type_Ent);
       StrLen           : constant Integer :=
                                     Integer (String_Length (Strval (N)));
       String_Length_Expr : constant Irep := New_Irep (I_Constant_Expr);
```

Comment:
Is this used anywhere?

Reply:
Technically not, at the moment - but I think it *will* be again when we finally get types for string literals sorted out. The gnat compiler is happy enough for the moment though as String_Length_Expr gets passed into Set_Type/Set_Value. Happy to comment out these as well if people prefer.

Reply:
I've commented out this currently unused code too.